utils/generate-catalog.sh

```diff
@@ -12,9 +12,10 @@ CHANNEL_NAME=preview
 OPM="${1?:Error \$OPM not set. Bye}"
 YQ="${2?:Error \$YQ not set. Bye}"
 BUNDLE_IMG="${3?:Error \$BUNDLE_IMG not set. Bye}"
-LIMITADOR_OPERATOR_BUNDLE_IMG="${4?:Error \$LIMITADOR_OPERATOR_BUNDLE_IMG not set. Bye}"
-AUTHORINO_OPERATOR_BUNDLE_IMG="${5?:Error \$AUTHORINO_OPERATOR_BUNDLE_IMG not set. Bye}"
-CATALOG_FILE="${6?:Error \$CATALOG_FILE not set. Bye}"
+REPLACES_VERSION="${4?:Error \$REPLACES_VERSION not set. Bye}"
```

Comment:
Almost worth using a `getopt` loop here to retrieve named args. Future improvement idea.